src/native/libunwind/CMakeLists.txt

```diff
@@ -0,0 +1,392 @@
+ensure_variable_set(LIBUNWIND_SOURCE_DIR)
+set(LIB_NAME unwind_xamarin)
+set(LIB_ALIAS xa::unwind)
+
+#
+# Read libunwind version
+#
+set(CONFIGURE_AC "${LIBUNWIND_SOURCE_DIR}/configure.ac")
+file(STRINGS "${CONFIGURE_AC}" UNWIND_MAJOR_VERSION_AC REGEX "^[ \t]*define[ \t]*\\([ \t]*pkg_major,[ \t]*(.*)\\)")
+file(STRINGS "${CONFIGURE_AC}" UNWIND_MINOR_VERSION_AC REGEX "^[ \t]*define[ \t]*\\([ \t]*pkg_minor,[ \t]*(.*)\\)")
+file(STRINGS "${CONFIGURE_AC}" UNWIND_EXTRA_VERSION_AC REGEX "^[ \t]*define[ \t]*\\([ \t]*pkg_extra,[ \t]*(.*)\\)")
+string(REGEX REPLACE "^[ \t]*define[ \t]*\\([ \t]*pkg_major,[ \t]*(.*)\\)" "\\1" UNWIND_MAJOR_VERSION "${UNWIND_MAJOR_VERSION_AC}")
+string(REGEX REPLACE "^[ \t]*define[ \t]*\\([ \t]*pkg_minor,[ \t]*(.*)\\)" "\\1" UNWIND_MINOR_VERSION "${UNWIND_MINOR_VERSION_AC}")
+string(REGEX REPLACE "^[ \t]*define[ \t]*\\([ \t]*pkg_extra,[ \t]*(.*)\\)" "\\1" UNWIND_EXTRA_VERSION "${UNWIND_EXTRA_VERSION_AC}")
+
+set(PKG_MAJOR "${UNWIND_MAJOR_VERSION}")
+set(PKG_MINOR "${UNWIND_MINOR_VERSION}")
+set(PKG_EXTRA "${UNWIND_EXTRA_VERSION}")
+set(PACKAGE_STRING "libunwind-xamarin")
+set(PACKAGE_BUGREPORT "")
```

Comment:
Why not set `PACKAGE_BUGREPORT`?

Reply:
It's set to override the bug report address in `libunwind`, we don't want to have their address in our code base.

Reply:
Let me rephrase: I'm not wondering why we're setting it at all.  I'm wondering why we're setting it to the empty string instead of github.com/xamarin/xamarin.android/issues.

Reply:
We're setting it to empty string because it sets bug report address for **libunwind**, not us - so we shouldn't put our address in their codebase (even though nobody's probably  going to see it there). Just a matter of keeping stuff tidy, I guess.